### PR TITLE
My story

### PR DESCRIPTION
**Pull request by brianquach**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2362)

### CODE REVIEW

File: english/pipboy/lockpick/lockpick_100.md [hunk 1]
```diff
@@ -4,5 +4,7 @@ Mo' problem.
 
 Door's open. You head into the vast wasteland.
 
+[go straight until...](../love/love.md)
```
Comment (casey-collab) [inline english/pipboy/lockpick/lockpick_100.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by brianquach** [comment]
Alright fixed, please review! @casey-collab

**Comment by casey-collab** [comment]
Looks great! :purple_heart:

**Pull request merged**